docs/language_guide.md

```diff
@@ -1129,32 +1222,44 @@ struct Person: Animal {
     return "Huh?"
   }
 
-  // Person can also have functions in addition to Animal
+  // Person can also have functions in addition to Animal.
   public func greet() -> String {
     return "Hi"
   }
 }
+```
+
+### Contract traits
+
+Traits can be declared for contracts using the syntax:
+
+```swift
+contract trait <trait-name> {
+  // trait members
+}
+```
+
+Contracts can conform to contract traits using the following syntax for their declaration part:
 
+```swift
+contract <contract-name>: <trait-1>, <trait-2>, ... {
+  // ...
+}
 ```
-### Contract Traits
-Traits can be implemented for structures using the keyword combination of `contract trait` followed by a unique identifier before the block of trait members.
 
-Contract traits can contain anonymous contract behaviour declarations containing methods or method signatures, or events.
+Contract traits can contain anonymous contract behaviour declarations containing functions, function signatures, and events.
```

Comment:
Methods was better considering these functions are tied to an instance

Reply:
& any other replacement of 'method' => 'function' that you did. Pls revert those thanx

Reply:
This was the only place methods was used in the entire language guide.